rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/reports/TabulatorGenerator.xtend

```diff
@@ -132,25 +169,25 @@ class TabulatorGenerator {
 	}
 	
 	def generate(IFileSystemAccess2 fsa, Function func) {
-		if (func.isProjection) {
-			val tabulatorClass = func.toProjectionTabulatorJavaClass
+		if (func.isFunctionTabulatable) {
+			val tabulatorClass = func.toApplicableTabulatorClass
 			val topScope = new JavaScope(tabulatorClass.packageName)
 			
 			val projectionType = typeProvider.getRTypeOfSymbol(func.output)
```

Comment:
This should be called `functionOutputType` not `projectionType`

Reply:
done